wasm/tests/crypto.test.ts

```diff
@@ -0,0 +1,138 @@
+// external references used for comparison
+import nacl from 'tweetnacl';
+import {sha256} from 'crypto-hash';
+import {performance} from 'perf_hooks';
+import * as crypto from 'crypto';
+
+import {waitReady, ed25519, hasher} from './../';
+
+const toHex = (arrayBuffer: ArrayBuffer | SharedArrayBuffer) => {
+  return Buffer.from(arrayBuffer).toString('hex');
+};
+
+const benchmark = (func: () => void, iterations: number) => {
+  const start = performance.now();
+  for (let i = 0; i < iterations; i++) {
+    func();
+  }
+  const finish = performance.now();
+  return finish - start;
+};
+
+describe('Benchmark in Node.js WASM implementation of ED25519 vs tweetnacl', () => {
+  beforeAll(async () => {
+    await waitReady();
+  });
+
+  // on avg. WASM is 20-25x faster than tweetnacl but I am testing conservative assumption
+  test('WASM sign method should be faster than nacl by at least 15x', () => {
```

Comment:
This is great for proving out why we should add wasm support, but I don't think we need to include it in this PR

Reply:
done